src/commands/deploy.ts

```diff
@@ -60,17 +58,9 @@ function handleError(
         > ${constructCommandName(fullCommand, 'deploy', ['--force'])} 
     `;
     logger.error(messageBody, err.message);
-  } else if (err.name === 'HTTPError') {
-    const responseBody = JSON.parse(
-      (err as HttpError).body
-    ) as ApiErrorResponse;
-    const messageBody = stripIndent`
-      ${responseBody.message}
-
-      More info: ${responseBody.more_info}
-    `;
-
-    logger.error(messageBody);
+  } else if (err.name === 'TwilioApiError') {
+    const apiError = err as ClientApiError;
```

Comment:
Why do you need to cast this error as `ClientApiError` but in the activate, list and logs commands you don't?